Code/RDBoost/Wrap.h

```diff
@@ -18,6 +18,10 @@
 //
 #include <boost/python.hpp>
 #include <boost/python/stl_iterator.hpp>
+#include <memory>
+// ha ha... we don't actually want to use this, but we need to keep support old
+// C++ versions:
+#define rdk_unique_ptr std::auto_ptr
```

Comment:
Note that auto_ptrs are deprecated for a reason - they have very weird transfer ownership semantics (which unique_ptr actually fixes with introduction of std::move)